vagrant.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Latest LTS as of Nov 22, 2016
+NODE_VERSION="6.9.1"
+
+# Install new nodejs.
+_node_filename="node-v${NODE_VERSION}-linux-x64.tar.xz"
+_node_url="https://nodejs.org/dist/v${NODE_VERSION}/${_node_filename}"
```

Comment:
`nvm` is great, but we don't really benefit too much from it in this case as we don't need to switch the versions of nodejs.

Besides, nodejs is only part of the problem. The still unsolved problem is the outdated rabbitmq. If we have a newer base box, we can simply install node and rabbitmq from the repo and both problems should be solved automatically.